### ISSUE TITLE
2.3.3 Animation from Interactions (Level AAA)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#549]
#### [2.3.3 Animation from Interactions](@@ URL for the SC in WCAG 2.2 @@)

(Level AAA)

>[Motion animation](https://www.w3.org/TR/WCAG22/#dfn-motion-animation) triggered by interaction can be disabled, unless the animation is [essential](https://www.w3.org/TR/WCAG22/#dfn-essential) to the functionality or the information being conveyed.

##### Applying SC 2.3.3 Animation from Interactions to Non-Web Documents and Software

This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.3.3](https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interactions.html#intent).

***
**For WCAG2ICT consideration:** Are there any notes needed or considerations for closed functionality?
***

**Comment by pday1** [comment]
I don't think that we need any additional notes for this.
Any closed functionality or system that has animation baked in (e.g. old legacy systems that might pre-render content and serve as a sequence of images or frames) would be covered under the "essential" exception, so nothing is needed for SC problematic for closed IMO.

Proposed content is therefore as follows. 


> ##### Applying SC 2.3.3 Animation from Interactions to non-web documents and non-web software
> 
> This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.3.3](https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interactions.html#intent).

**Comment by bruce-usab** [comment]
> I don't think that we need any additional notes for this.  …nothing is needed for SC problematic for closed IMO.

Agreed, and as per call today, we all concurred.

> Any closed functionality or system that has animation baked in…

@pday1 — I just wanted to note that I do _not_ agree that this particular example is covered under the “essential” exception.  Maybe I should (re)reread our [comments  on closed functionality](https://w3c.github.io/wcag2ict/#comments-on-closed-functionality)?

**Comment by pday1** [comment]
As per TF meeting of 23 April 2025, applies as written.
[For 2.3.3 Animation from Interactions,incorporate proposal into the editor’s draft, as is](https://www.w3.org/2026/04/23-wcag2ict-minutes.html#e3b1)

Proposed content is therefore as follows. 

> ##### Applying SC 2.3.3 Animation from Interactions to non-web documents and non-web software
> 
> This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.3.3](https://www.w3.org/WAI/WCAG22/Understanding/animation-from-interactions.html#intent).

**Comment by pday1** [comment]
> > Any closed functionality or system that has animation baked in…
> 
> [@pday1](https://github.com/pday1) — I just wanted to note that I do _not_ agree that this particular example is covered under the “essential” exception. Maybe I should (re)reread our [comments on closed functionality](https://w3c.github.io/wcag2ict/#comments-on-closed-functionality)?

Thanks @bruce-usab - I do think that we may have to do another pass on comments on closed functionality, and the SC problematic for closed functionality sections once all level AAA SCs are populated and read it all in context to see if there are things that are missing, or just inconsistent between SCs. Shout if you think we do need a note here.

**Comment by pday1** [comment]
In PR https://github.com/w3c/wcag2ict/pull/906
